src/kata3.numberToReversedDigits.js

```diff
@@ -1,5 +1,7 @@
 const numberToReversedDigits = (number) => {
-
+   const numberArray = Array.from(number.toString()).map(Number)
```

Comment:
`number.toString().split('').map()` would also work. For the sake of readability I might make sense to split this over a few lines.